pkg/fanal/analyzer/language/nodejs/license/license.go

```diff
@@ -0,0 +1,110 @@
+package license
+
+import (
+	"errors"
+	"io/fs"
+	"path"
+	"strings"
+
+	"golang.org/x/xerrors"
+
+	dio "github.com/aquasecurity/go-dep-parser/pkg/io"
+	"github.com/aquasecurity/go-dep-parser/pkg/nodejs/packagejson"
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer/language/nodejs"
+	"github.com/aquasecurity/trivy/pkg/fanal/types"
+	"github.com/aquasecurity/trivy/pkg/licensing"
+	"github.com/aquasecurity/trivy/pkg/log"
+	"github.com/aquasecurity/trivy/pkg/utils/fsutils"
+)
+
+func ParseLicenses(
+	packageJsonParser *packagejson.Parser,
+	classifierConfidenceLevel float64,
+	licenses map[string][]string,
+) func(fsys fs.FS, root string) error {
+	return func(fsys fs.FS, root string) error {
+		if fsys == nil {
+			return xerrors.New("fs.FS required")
+		}
+
+		walkDirFunc := func(pkgJsonPath string, d fs.DirEntry, r dio.ReadSeekerAt) error {
+
+			pkg, err := packageJsonParser.Parse(r)
+			if err != nil {
+				return xerrors.Errorf("unable to parse %q: %w", pkgJsonPath, err)
+			}
+
+			ok, licenseFileName := isLicenseRefToFile(pkg.License)
+
+			if !ok {
+				licenses[pkg.ID] = []string{pkg.License}
+				return nil
+			}
+
+			log.Logger.Debugf("Licenses are missing in %q, an attempt to find them in the LICENSE file", pkgJsonPath)
```

Comment:
I understood that it can confused - users can think that we only parse `LICENSE` file.
```suggestion
			log.Logger.Debugf("License names are missing in %q, an attempt to find them in the%q file", pkgJsonPath, licenseFileName)
```

Reply:
fixed